scripts/utils/constants.ts

```diff
@@ -11,6 +11,7 @@ export const STRIKE_STEP = {
   BADGER: 0.1 * 1e8,
   BAL: 0.1 * 1e8,
   SPELL: 1000, // 0.00001 * 1e8
+  SAMB: 1000, // 0.00001 * 1e8
```

Comment:
Will we be using this strike step? I just copied SPELL's for now, but more generally how do we choose the strike step to use?

Reply:
lets use 100000, // 0.001 * 1e8

Reply:
Addressed in [e34465b](https://github.com/ribbon-finance/ribbon-v2/pull/293/commits/e34465b30162c78b1c9313d3468d42cd1c325bc4). How did you decide what to use actually?

Reply:
based on the price of the asset, we decide a reasonable increment. for example if asset is $100, we might increment in intervals of $1 for optimal granularity of strike price. if asset is $10, we might use $0.1.